.github/workflows/post-integration.yml

```diff
@@ -40,11 +40,14 @@ jobs:
         with:
           java-version: 1.11
 
-      - name: 📋 Prepare build for unit tests
-        run: |
-          dotnet clean -c Debug
-          dotnet restore
-          dotnet build -c Debug /p:UseSourceLink=true --no-restore
+      - name: 🧹 Clean
+        run: dotnet clean -c Release && dotnet nuget locals all --clear
```

Comment:
Why is this needed. Each run of a workflow gets a clean file system to work with, as far as I know.

Reply:
Apparently it does not. We had alot of issues with this.